kitsune/search/v2/documents.py

```diff
@@ -15,33 +15,27 @@
 
 
 class WikiDocument(SumoDocument):
-    url = field.Keyword()
     updated = field.Date()
 
     product_ids = field.Keyword(multi=True)
     topic_ids = field.Keyword(multi=True)
-
-    # Document specific fields (locale aware)
-    title = SumoLocaleAwareTextField()
-    content = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
-    summary = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
-    keywords = SumoLocaleAwareTextField(multi=True)
-
     locale = field.Keyword()
-    current_id = field.Integer()
-    parent_id = field.Integer()
+    parent_id = field.Keyword()
     category = field.Keyword()
     slug = field.Keyword()
     is_archived = field.Boolean()
     recent_helpful_votes = field.Integer()
 
+    # Document specific fields (locale aware)
+    title = SumoLocaleAwareTextField()
+    content = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
+    summary = SumoLocaleAwareTextField(store=True, term_vector="with_positions_offsets")
+    keywords = SumoLocaleAwareKeywordField(multi=True)
```

Comment:
do we want to normalize this? https://www.elastic.co/guide/en/elasticsearch/reference/current/normalizer.html

I think in our `prepare_keywords` method we want to split on white-space, so `keyword-1 keyword-2 keyword-3` becomes [`keyword-1`, `keyword-2`, `keyword-3`], otherwise I don't think we can match on `keyword-2` for instance without using wildcards either side.

Reply:
Missed that bit from the other PR. Will update. This is the prepare method that I will be using here:
https://github.com/mozilla/kitsune/pull/4560/files#diff-1701974c94f568a36fc78cde0bb414faa0963f61133dc8b4c9627c69071fd560R61